TACC/Frontera-Portal
# Task/FP-779 remove onboarding middleware

## Overview: ##
remove onboarding middleware

## PR Status: ##

## Related Jira tickets: ##

* [FP-779](https://jira.tacc.utexas.edu/browse/FP-779)

## Summary of Changes: ##
* Remove onboarding middleware

## Testing Steps: ##
1. Logout of portal
2. Go to cep.dev/workbench/dashboard
3. After logging in, check that you are routed to destination in step 2.

## Notes: ##
@jchuahtacc  pointed out during a different PR review of a that the middleware is really no longer used.  Removing now as the check of user.profile is unneeded as that is ensured during user authentification.
